config/authorization/config.ex

```diff
@@ -58,22 +58,46 @@ defmodule Acl.UserGroups.Config do
         name: "public",
         useage: [:read],
         access: %AlwaysAccessible{}, 
-        graphs: [ %GraphSpec{
+        graphs: [
+          %GraphSpec{
+                    graph: "http://mu.semte.ch/graphs/mow/registry",
+                    constraint: %ResourceConstraint{
+                      resource_types: [
+                        "http://www.w3.org/2004/02/skos/core#ConceptScheme",
+                        "https://data.vlaanderen.be/ns/mobiliteit#Verkeersbordconcept",
+                        "https://data.vlaanderen.be/ns/mobiliteit#Verkeersbordcategorie",
+                        "https://data.vlaanderen.be/ns/mobiliteit#VerkeersbordconceptStatus",
+                        "https://data.vlaanderen.be/ns/mobiliteit#Wegmarkeringconcept",
+                        "https://data.vlaanderen.be/ns/mobiliteit#Verkeerslichtconcept",
+                        "http://xmlns.com/foaf/0.1/Image",
+                        "http://data.lblod.info/vocabularies/mobiliteit/VerkeersbordconceptStatusCode",
+                        "http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#FileDataObject",
+                        "http://www.w3.org/2000/01/rdf-schema#Resource",
+                        "http://www.w3.org/ns/shacl#Shape",
+                        "http://www.w3.org/ns/shacl#PropertyShape",
+                        "http://www.w3.org/ns/shacl#NodeShape",
+                        "http://mu.semte.ch/vocabularies/ext/Template",
+                        "http://mu.semte.ch/vocabularies/ext/Mapping",
+                        "http://mu.semte.ch/vocabularies/ext/Relation",
+                        "http://mu.semte.ch/vocabularies/ext/CanBeCombinedWithRelation",
+                        "http://mu.semte.ch/vocabularies/ext/MustUseRelation",
+                        "http://mu.semte.ch/vocabularies/ext/Concept",
+                        "http://mu.semte.ch/vocabularies/ext/Resource",
+                      ] } },
+          %GraphSpec{
                     graph: "http://mu.semte.ch/graphs/public",
                     constraint: %ResourceConstraint{
                       resource_types: [
-                        "http://data.vlaanderen.be/ns/besluit#Bestuurseenheid",
-                        "http://xmlns.com/foaf/0.1/Person",
-                        "http://xmlns.com/foaf/0.1/OnlineAccount"
+                        "http://data.vlaanderen.be/ns/besluit#Bestuurseenheid"
                       ]
                     } },
-                  %GraphSpec{
+          %GraphSpec{
```

Comment:
I don't think we actually need the session graph here, though I guess it can't hurt.

Reply:
Indeed it's empty, will remove it :)